easybuild/easyblocks/c/crisprdav.py

```diff
@@ -0,0 +1,100 @@
+##
+# Copyright 2020-2021 Ghent University
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/easybuilders/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+EasyBuild support for building and installing Metagenome-Atlas, implemented as an easyblock.
+
+@author: Denis Kristak (INUITS)
+"""
+import os
+from easybuild.tools.filetools import write_file
+from easybuild.easyblocks.generic.binary import Binary
+
+
+class EB_crisprdav(Binary):
+    """
+    Support for building/installing crispr-dav.
+    """
+
+    def post_install_step(self):
+        """Create config.txt files"""
+        crisprdav_installdir_ex = os.path.join(self.installdir, 'Examples/example1/')
+        abra2_dir = os.environ['EBROOTABRA2']
+        prinseq_dir = os.environ['EBROOTPRINSEQ']
+        flash_dir = os.environ['EBROOTFLASH']
```

Comment:
It's better to use the `get_software_root` function here, and you should also *check* that the result is not `None`, which verifies that these are indeed included as dependencies.